### DISCUSSION TITLE
killing gradio

### DISCUSSION THREAD

**Discussion by Yavuzhan-Baykara** [FurkanGozukara/Stable-Diffusion#41] [Show and tell]
!./gui.sh --share --headless
After running this line in Kaggle, it says 'gradio killing'. What should I do?

Output:
12:22:17-013271 INFO     Version: v22.3.1                                       
                                                                                
12:22:17-022372 INFO     nVidia toolkit detected                                
12:22:20-155258 INFO     Torch 2.0.1+cu118                                      
12:22:20-245089 INFO     Torch backend: nVidia CUDA 11.8 cuDNN 8700             
12:22:20-268267 INFO     Torch detected GPU: Tesla T4 VRAM 15110 Arch (7, 5)    
                         Cores 40                                               
12:22:20-270009 INFO     Verifying modules installation status from             
                         /kaggle/working/kohya_ss/requirements_linux.txt...     
12:22:20-275483 INFO     Verifying modules installation status from             
                         requirements.txt...                                    
12:22:24-295456 INFO     headless: True                                         
12:22:24-301090 INFO     Load CSS...                                            
Running on local URL:  http://127.0.0.1:7860/
Running on public URL: https://069705f6b04c640bce.gradio.live/

This share link expires in 72 hours. For free permanent hosting and GPU upgrades, run `gradio deploy` from Terminal to deploy to Spaces (https://huggingface.co/spaces)
^C
Keyboard interruption in main thread... closing server.
Killing tunnel 127.0.0.1:7860 <> https://069705f6b04c640bce.gradio.live/

**Comment by FurkanGozukara** [comment]
It is true. Kaggle banned it. A working way is only shared in our Patreon notebook atm : https://www.patreon.com/posts/kohya-sdxl-lora-88397937

new video : https://youtu.be/16-b1AjvyBE

**Reply by Yavuzhan-Baykara** [reply]
I thought so :) sensei,  I really liked your videos and will support you on pateon soon. cya

**Reply by FurkanGozukara** [reply]
thank you so much
